applications/nrf_desktop/README.rst

```diff
@@ -696,6 +808,13 @@ Hardware switch
 
 Precision Aim button
     * The button is located on the left side of the mouse, in the thumb area.
+
+      .. figure:: /images/nrf_desktop_gaming_mouse_top.svg
```

Comment:
**Image edit:**
Please remove the debug caption from this image.

@pdunaj , do we need the mention of LED 1 on this image?
I think it should be OK to keep it, but without "- peer state", and with an explanation in the docs about this LED 1.